src/main.rs

```diff
@@ -139,8 +140,9 @@ fn fs_prep(pid: unistd::Pid) -> String {
     rootfs
 }
 
-fn net_prep(/* pid, link, ip */) {
-    // network setup
+fn net_prep(ip: String, pid: u32) {
+    net::prep_network_stack();
+    net::set_outside_network(ip.to_string(), pid);
```

Comment:
`ip` is already of type `String`
```suggestion
    net::set_outside_network(ip, pid);
```

Reply:
get this err 
```
mismatched types
expected struct `String`, found `&str`
```
don't know why it occur.

Reply:
Fixed: 3da03d3